src/MicrosoftTeams.ts

```diff
@@ -21,14 +21,26 @@ namespace microsoftTeams {
 
   const version = "1.2";
 
-  const validOrigins = [
-    "https://teams.microsoft.com",
-    "https://teams.microsoft.us",
-    "https://int.teams.microsoft.com",
-    "https://devspaces.skype.com",
-    "https://ssauth.skype.com",
-    "http://dev.local" // local development
-  ];
+  const validHostRegEx = new RegExp(
```

Comment:
Can we name this back to validOriginsRegExp, since it's actually checking the origin, not just the host?

Reply:
sure